app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -307,6 +308,15 @@ export default {
           this.$emit('scrollToMessage');
         });
     },
+
+    autoExpandTextarea() {
+      const textarea = document.querySelector('.reply-box textarea');
```

Comment:
I guess instead of querySelector, we should use refs. Here textarea already has a ref attached to it (`this.$refs.messageInput`). Please make use of that.